packages/block-editor/src/components/block-edit/multiple-usage-warning.js

```diff
@@ -24,17 +24,15 @@ export function MultipleUsageWarning( {
 		<Warning
 			actions={ [
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
This occurs when a block has multiple supports set to false in block.json and you add that block multiple times in the editor.

| Before | After     |
| :-------- | :------- |
| ![multiple usage warning 36](https://github.com/user-attachments/assets/ee5ba0d3-85cf-4658-9c7a-0eede0f8fb70)| ![multiple usage warning 40px](https://github.com/user-attachments/assets/5aec9433-6622-4bd7-b376-0abd37865ba3)|